modules/gapi/test/cpu/gapi_core_tests_fluid.cpp

```diff
@@ -285,260 +285,4 @@ INSTANTIATE_TEST_CASE_P(CopyTestFluid, CopyTest,
-                                       cv::Size(640, 480),
-                                       cv::Size(128, 128)),
-/*init output matrices or not*/ testing::Bool()));
-
-INSTANTIATE_TEST_CASE_P(FlipTestCPU, FlipTest,
-                        Combine(Values(CV_8UC1, CV_16UC1, CV_16SC1),
-                                Values(0,1,-1),
-                                Values(cv::Size(1280, 720),
-                                       cv::Size(640, 480),
-                                       cv::Size(128, 128)),
-/*init output matrices or not*/ testing::Bool()));
-
-INSTANTIATE_TEST_CASE_P(CropTestCPU, CropTest,
-                        Combine(Values(CV_8UC1, CV_16UC1, CV_16SC1),
-                                Values(cv::Rect(10, 8, 20, 35), cv::Rect(4, 10, 37, 50)),
-                                Values(cv::Size(1280, 720),
-                                       cv::Size(640, 480),
-                                       cv::Size(128, 128)),
-/*init output matrices or not*/ testing::Bool()));
-
-INSTANTIATE_TEST_CASE_P(LUTTestCPU, LUTTest,
-                        Combine(Values(CV_8UC1, CV_8UC3),
-                                Values(CV_8UC1),
-                                Values(cv::Size(1280, 720),
-                                       cv::Size(640, 480),
-                                       cv::Size(128, 128)),
-/*init output matrices or not*/ Values(true)));
-
-INSTANTIATE_TEST_CASE_P(LUTTestCustomCPU, LUTTest,
-                        Combine(Values(CV_8UC3),
-                                Values(CV_8UC3),
-                                Values(cv::Size(1280, 720),
-                                       cv::Size(640, 480),
-                                       cv::Size(128, 128)),
-/*init output matrices or not*/ Values(true)));
-
-// PLEASE DO NOT PUT NEW ACCURACY TESTS BELOW THIS POINT! //////////////////////
-
-INSTANTIATE_TEST_CASE_P(ConvertToCPU, ConvertToTest,
-                        Combine(Values(CV_8UC3, CV_8UC1, CV_16UC1, CV_32FC1),
-                                Values(CV_8U, CV_16U, CV_32F),
-                                Values(cv::Size(1280, 720),
-                                       cv::Size(640, 480),
-                                       cv::Size(128, 128))));
-
-INSTANTIATE_TEST_CASE_P(ConcatHorTestCPU, ConcatHorTest,
-                        Combine(Values(CV_8UC1, CV_16UC1, CV_16SC1),
-                                Values(cv::Size(1280, 720),
-                                       cv::Size(640, 480),
-                                       cv::Size(128, 128))));
-INSTANTIATE_TEST_CASE_P(ConcatVertTestCPU, ConcatVertTest,
-                        Combine(Values(CV_8UC1, CV_16UC1, CV_16SC1),
-                                Values(cv::Size(1280, 720),
-                                       cv::Size(640, 480),
-                                       cv::Size(128, 128))));
-
-//----------------------------------------------------------------------
```

Comment:
So do we really have all these tests running with Fluid now?

Reply:
Added unused tests. Fixed.